packages/remix-dev/compiler/utils/postcss.ts

```diff
@@ -64,7 +64,14 @@ export async function loadPostcssPlugins({
 
       plugins.push(...postcssConfig.plugins);
     } catch (err) {
-      // If they don't have a PostCSS config, just ignore it.
+      // If they don't have a PostCSS config, just ignore it,
+      // otherwise rethrow the error.
+      if (
+        err instanceof Error &&
+        !/No PostCSS Config found/i.test(err.message)
```

Comment:
Is this an error we throw or a dependency throws?  The raw string matching feels somewhat fragile but if it's not our error then we may not have a choice.  If it is our error - we could move to a more explicit check?

Reply:
Yep, this isn't our error. I'm not a fan of this either, but the alternative would be to manually check for a PostCSS config ourselves before trying to load the config. This would mean checking for the following files:

- `package.json` (`postcss` field)
- `.postcssrc`
- `.postcssrc.json`
- `.postcssrc.yml`
- `.postcssrc.js`
- `.postcssrc.mjs`
- `.postcssrc.cjs`
- `.postcssrc.ts`
- `postcss.config.js`
- `postcss.config.mjs`
- `postcss.config.cjs`
- `postcss.config.ts`

While we could do this, this feels fragile too since PostCSS could change their config resolution logic in the future and we'd be hard-coding the current state.

We could improve this with a change upstream to allow ignoring missing config, but I didn't want that to block this fix.

Reply:
👍 makes sense to me!